<?xml version="1.0"?>
<launch>
	<group ns="rover">
		<node name="f9p" pkg="ublox" type="ublox_ros">
			<param name="serial_port" value="/dev/ttyACM0"/>
			<param name="rtk_type" value="2"/>
 			<param name="local_host" value="192.168.0.202"/>
			<param name="local_port" value="16145"/>
			<param name="remote_host" value="192.168.0.111"/>
			<param name="remote_port" value="16140"/>
			<param name="log_filename" value="../data/rod/redo_rod/one/base_binary"/>
		</node> 
		<node pkg="rosbag" type="record" name="record" output="screen" args="-O ../data/08_22_walk/rover3.bag --all"/>
	</group>
</launch>
